### ISSUE TITLE
Ampersands being escaped during XML transformation

### ISSUE THREAD

**Issue by dibble-james** [sergeyzwezdin/magic-chunks#16]
I have the joy of working with some EDMX models that use connection strings that I am looking to transform.  The connection string ends up looking something like:

`<add name="Connection" connectionString="metadata=res://*/Model.csdl|res://*/Model.ssdl|res://*/Model.msl;provider=System.Data.SqlClient;provider connection string=&quot;data source=server\instance;initial catalog=database;integrated security=True;multipleactiveresultsets=True;&quot;" providerName="System.Data.EntityClient" />`

The thing of note is the `&quot;` in there which is unavoidable.

So in VSTS I've created a variable with the new connection string which looks something like:

`metadata=res://*/Model.csdl|res://*/Model.ssdl|res://*/Model.msl;provider=System.Data.SqlClient;provider connection string=&quot;data source=other-server\instance;initial catalog=database;integrated security=True;multipleactiveresultsets=True;&quot;`

When transformed into the web.config it gets converted to:

`metadata=res://*/Model.csdl|res://*/Model.ssdl|res://*/Model.msl;provider=System.Data.SqlClient;provider connection string=&amp;quot;data source=other-server\instance;initial catalog=database;integrated security=True;multipleactiveresultsets=True;&amp;quot;`

**Comment by sergeyzwezdin** [comment]
`XElement` escaping values during document update, so you don't need to escape value by yorself.
In your case [try](https://github.com/sergeyzwezdin/magic-chunks/commit/e2a28f46640e01b6241e061b8d3830a6f1d92d90#diff-9c6eb11dca2f416c517f205c905045fdR284) to use:

```
metadata=res://*/Model.csdl|res://*/Model.ssdl|res://*/Model.msl;provider=System.Data.SqlClient;provider connection string="data source=other-server\instance;initial catalog=database;integrated security=True;multipleactiveresultsets=True;"
```

instead of 

```
metadata=res://*/Model.csdl|res://*/Model.ssdl|res://*/Model.msl;provider=System.Data.SqlClient;provider connection string=&quot;data source=other-server\instance;initial catalog=database;integrated security=True;multipleactiveresultsets=True;&quot;
```

Also, I've added fix which allows you to have escaped value. Will be shipped with `1.2` later.

**Comment by sergeyzwezdin** [comment]
Build `1.2` has been shipped.
